### DISCUSSION TITLE
Share with QR code

### DISCUSSION THREAD

**Discussion by devemux86** [devemux86/cruiser#291] [Q&A] [answered]
THAT would be a dream! if you could transfer the toures and the settings via a QR code.
We drive in small groups, or in the large group - only 2-3 people with Navigation in operation. We all have 6.5" . . . 6.8" Samsung mobile phones - one setting would always fit.
Saving a default setting with QR code . . . that would be very, very nice.

_Originally posted by @SF4User in https://github.com/devemux86/cruiser/discussions/290#discussioncomment-6791786._

**Comment by devemux86** [comment]
@SF4User thanks for the interesting idea!

Using QR codes to share routes, favorites, profiles can make the life of users easier,
without the difficulty of transferring files.

**Comment by devemux86** [comment]
The idea is probably to synchronize 2 devices with Cruiser:
- Share something with QR code with 1st Cruiser
- Scan QR code in other device, opens 2nd Cruiser
- Both Cruiser have the same shared content

**Comment by devemux86** [comment, marked as answer]
QR codes is an interesting idea, but they seem to have a small size limit.
They can not encode large data? As files with routes, favorites or profiles.

They seem more suitable for small texts, like links.

**Comment by WolfgangZ42** [comment]
For years I have stored all relevant data of a tour in a Dropbox directory, which I make available to all participants by a link. 
See my post in the Kurviger Forum from May 2021 - https://forum.kurviger.de/t/access-web-route-from-app/4925/9 

So everyone has access to the current status of all routes (either in .kurviger or .gpx format). No one needs to have a Dropbox userid, a simple download will bring all data into the download directory of the smartphone - as a .zip file. Easy to edit with file manager plus. If you want to try it out - an example: https://www.dropbox.com/scl/fo/vg47kbtgexy3sxukh8x21/h?rlkey=1ppj5cx29oqx4qnjpf43tlsyw&dl=0 

Whether I have to pass on this link as QR, I dare to doubt.

**Reply by SF4User** [reply]
Ja! Im Prinzip genau der work around, den ich/wir schon bei Kurviger#2 angewandt haben. Da habe ich meine Google Drive freigeschaltet und als Backup genutzt. Wer mit dem Daten hin- und her schubsen kein Problem hat, sehr praktikabel.
Eine Erfahrung hatten wir die Tage in Italien gemacht. Tag#1 bis Tag#4 war über die Google Drive verteilt, easy going . . . am Tag#5 mußten wir wegen Wetter unterwegs umdisponieren. Da wäre es praktisch gewesen, wenn wir im Feld eben mal plop . . . plop . . . plop einen QR-Code der Tour verteilen hätten können. Wenn schon das Gewitter aufzieht, dann zählt halt jede Sekunde 😳 😱. 
Ist immer mal ein Kumpel dabei, der keinen Datei Manager auf dem Handy installiert hat . . .  Wir reden über ein Komfort Problem . . . 👍👍
